interfaces/tempo_cluster/v0/schema.py

```diff
@@ -0,0 +1,78 @@
+"""This file defines the schemas for the provider and requirer sides of this relation interface.
+
+It must expose two interfaces.schema_base.DataBagSchema subclasses called:
+- ProviderSchema
+- RequirerSchema
+"""
+from enum import Enum
+from typing import Optional, Dict, Any, Literal
+
+from interface_tester.schema_base import DataBagSchema
+from pydantic import BaseModel
+
+ReceiverProtocol = Literal[
+    "zipkin",
+    "otlp_grpc",
+    "otlp_http",
+]
+
+class TempoClusterProviderAppData(BaseModel):
+    """TempoClusterProviderAppData."""
+    tempo_config: Dict[str, Any]
+    loki_endpoints: Optional[Dict[str, str]] = None
+    ca_cert: Optional[str] = None
+    server_cert: Optional[str] = None
+    privkey_secret_id: Optional[str] = None
```

Comment:
I'm wondering if we should describe a bit more what all the fields other than `tempo_config` are. In this form it might be a bit unclear.

Reply:
+1, will add a description